cf-agent/verify_packages.c

```diff
@@ -3289,7 +3289,9 @@ bool ExecPackageCommand(EvalContext *ctx, char *command, int verify, int setCmdC
             }
         }
 
-        ReplaceStr(line, lineSafe, sizeof(lineSafe), "%", "%%");
+        char lineSafe[line_size * 2];
```

Comment:
Shouldn't this be `2 * res` if we wanted to make it actually safe?

Reply:
Yes, you are mostly correct, although it should be `(res + 1) * 2`, since `res` does not contain `\0`

Reply:
...or `res * 2 + 1` is better, actually.